archive-workitem/src/main/java/org/jbpm/process/workitem/archive/ArchiveWorkItemHandler.java

```diff
@@ -36,11 +36,13 @@
 @Wid(widfile = "ArchiveDefinitions.wid", name = "Archive",
         displayName = "Archive",
         defaultHandler = "mvel: new org.jbpm.process.workitem.archive.ArchiveWorkItemHandler()",
+        documentation = "${artifactId}/index.html",
         parameters = {
                 @WidParameter(name = "Archive"),
                 @WidParameter(name = "Files")
         },
         mavenDepends = {
+                @WidMavenDepends(group = "${groupId}", artifact = "${artifactId}", version = "${version}"),
                 @WidMavenDepends(group = "org.apache.commons", artifact = "commons-compress", version = "1.12")
```

Comment:
I don't think the org.apache.commons... is needed here as it is already a compile time dependency of archive-workitem module so will already be included.